app/models/item.rb

```diff
@@ -7,4 +7,11 @@ class Item < ActiveRecord::Base
   validates :region, presence: true
   validates :eve_type, presence: true
   validates :info_date, presence: true
+
+  include PgSearch
+  pg_search_scope :search,
+  against: :name,
+  using: {
+  tsearch: { prefix: true }
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.